docs/framework/performance/garbage-collection-etw-events.md

```diff
@@ -108,6 +110,41 @@ The following table shows the event data:
 |GCHandleCount|win:UInt32|The number of garbage collection handles in use.|
 |ClrInstanceID|win:UInt16|Unique ID for the instance of CLR or CoreCLR.|
   
+## GCHeapStats_V2 Event
+
+The following table shows the keyword and level:
+
+|Keyword for raising the event|Level|
+|-----------------------------------|-----------|
+|`GCKeyword` (0x1)|Informational (4)|
+
+The following table shows the event information:
+
+|Event|Event ID|Description|
+|-----------|--------------|-----------------|
+|`GCHeapStats_V2`|4|Shows the heap statistics at the end of each garbage collection.|
+
+The following table shows the event data:
+
+|Field name|Data type|Description|
+|----------------|---------------|-----------------|
+|GenerationSize0|win:UInt64|The size, in bytes, of generation 0 memory.|
+|TotalPromotedSize0|win:UInt64|The number of bytes that are promoted from generation 0 to generation 1.|
+|GenerationSize1|win:UInt64|The size, in bytes, of generation 1 memory.|
+|TotalPromotedSize1|win:UInt64|The number of bytes that are promoted from generation 1 to generation 2.|
+|GenerationSize2|win:UInt64|The size, in bytes, of generation 2 memory.|
+|TotalPromotedSize2|win:UInt64|The number of bytes that survived in generation 2 after the last collection.|
+|GenerationSize3|win:UInt64|The size, in bytes, of the large object heap.|
+|TotalPromotedSize3|win:UInt64|The number of bytes that survived in the large object heap after the last collection.|
+|FinalizationPromotedSize|win:UInt64|The total size, in bytes, of the objects that are ready for finalization.|
+|FinalizationPromotedCount|win:UInt64|The number of objects that are ready for finalization.|
+|PinnedObjectCount|win:UInt32|The number of pinned (unmovable) objects.|
+|SinkBlockCount|win:UInt32|The number of synchronization blocks in use.|
```

Comment:
@davmason 
Should rather be `SyncBlockCount`, but from looking at the code looks like it the name is cut in stone
https://github.com/dotnet/runtime/blob/921bca0ae0d07a7dfd83d5d70eaf49759ab8463b/src/coreclr/vm/gctoclreventsink.h#L29